### PR TITLE
Update all Dockerfile to check multiple key servers on failure

### PR DESCRIPTION
**Pull request by goller**

### CODE REVIEW

File: chronograf/1.3/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,13 @@
 FROM buildpack-deps:jessie-curl
 
-RUN gpg \
-    --keyserver hkp://ha.pool.sks-keyservers.net \
-    --recv-keys 05CE15085FC09D18E99EFB22684A14CF2582E0C5
+RUN set -ex \
+  && for key in \
```
Comment (jsternberg) [inline chronograf/1.3/Dockerfile]: Can you put the `&&` at the end of the line like the other areas and space this to match the other formatting?

Comment (goller) [inline chronograf/1.3/Dockerfile]: @jsternberg fixed in 210b52e

File: chronograf/1.3/alpine/Dockerfile [hunk 2]
```diff
@@ -6,9 +6,15 @@ RUN apk add --no-cache ca-certificates && \
...
 ENV CHRONOGRAF_VERSION 1.3.0
 
-RUN apk add --no-cache --virtual .build-deps wget gnupg tar && \
-    gpg --keyserver hkp://ha.pool.sks-keyservers.net \
-        --recv-keys 05CE15085FC09D18E99EFB22684A14CF2582E0C5 && \
+RUN set -ex && \
+    apk add --no-cache --virtual .build-deps wget gnupg tar && \
+    for key in \
+    05CE15085FC09D18E99EFB22684A14CF2582E0C5 \
+    ; do \
```
Comment (jsternberg) [inline chronograf/1.3/alpine/Dockerfile]: Can you make the style here consistent between the normal and the alpine images? I prefer the current layout from the normal images to the format the alpine ones have, but consistency is key.

Comment (goller) [inline chronograf/1.3/alpine/Dockerfile]: Fixed in e22f6a0

**Pull request merged**